src/components/sokperson/SokContainer.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { NavigationBar } from '@/components/NavigationBar';
+import SokPerson from '@/components/sokperson/SokPerson';
+
+export default function SokContainer() {
+  return (
```

Comment:
La først på amplitude-logging for besøk her (som vi har i `OversiktContainer`), men det ser ut som amplitude logger page view ut av boksen. Så vi logger muligens noe dobbelt i `OversiktContainer`?

Reply:
Hmm, altså `besøk`-handlingen i amplitude logges automatisk?

Reply:
Amplitude logger automatisk en event med type `[Amplitude] Page Viewed` som høres ut som i praksis det samme som `besøk`-eventen?

Reply:
Ja det kan stemme - med `besøk` så kan vi vel definere datafeltene selv, mens den innebygde sikkert er forhåndsbestemt. De funker sikkert like bra 🙌🏼